File: ulohy/04-velikost-pismene/reseni/main.cpp
#include <iostream>
#include <string>

int main() {
    char letter;
    std::cin >> letter;

    std::string output;

    // Místo čísel lze použít i přímo chary 'a' apod.
    if (letter >= 65 && letter <= 90) {
        output = "upper";
    } else if (letter >= 97 && letter <= 122) {
        output = "lower";
    }

    std::cout << output << std::endl;

    return 0;
}

File: ulohy/04-velikost-pismene/reseni/main.java
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String input = scanner.nextLine();
        char letter = input.charAt(0);

        String output = "";

        // Místo čísel lze použít i přímo chary 'a' apod.
        if (letter >= 65 && letter <= 90) {
            output = "upper";
        } else if (letter >= 97 && letter <= 122) {
            output = "lower";
        }

        System.out.println(output);
    }
}
